Task30 consistency

tzielin1 opened the pull request:
Changed all instances of listOfIdeals to ideals in functions and documentation. Should we also do this in the tests?
Are there any other consistencies you can think of? They seem to be covered now with this and Task 27.

tzielin1 committed:
Changed listOfIdeals to ideals in functions and Documentation

tzielin1 committed:
loading fixes

Sondzus commented:
Thanks @tzielin1 . I think this is fine; and I don't think we have to worry about changing these in the tests. I'll merge this now.

Sondzus committed:
Merge branch 'master' into TASK30-consistency

sonjapetrovic committed:
master merge clean-up, part 2.

the CMStats test fails but I don’t want to fix it because there’s
another branch dealing with CM stats output.

monicayun committed:
Merge branch 'master' into TASK30-consistency

# Conflicts:
#	RandomMonomialIdeals.m2
made verbose message in idealsFromGeneratingSets the same as the rest

monicayun committed:
change verbose message for regStats

took out  concatenate
